ttnn/cpp/pybind11/activation.hpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <pybind11/operators.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "tt_eager/tensor/tensor.hpp"
+#include "ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_op.hpp"
+#include "export_enum.hpp"
+
+namespace py = pybind11;
+
+namespace ttnn {
+namespace activation {
+void py_module(py::module& m) {
+    using namespace ttnn::operations::unary;
+
+    // Export the UnaryOpType enum using the export_enum utility
+    export_enum<UnaryOpType>(m, "ActivationUnaryOpType");
+
+    // Export the UnaryWithParam class
+    py::class_<UnaryWithParam>(m, "ActivationUnaryWithParam")
```

Comment:
please rename to `ActivationUnaryWithParam ` from `UnaryWithParam`

Reply:
I'll rename the class to "UnaryWithParam" same as the struct in ttnn/unary it refers to. is that ok ?

Reply:
<img width="938" alt="image" src="https://github.com/user-attachments/assets/7af6535a-f4a6-404d-818b-a6b456305f81">